src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const DayRentCost = 40;
+  const ThreeDayBonus = 20;
+  const FiveDayBonus = 50;
+
+  if (days <= 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return (DayRentCost * days);
+  }
 
+  if (((days > 3) || (days === 3)) && (days < 7)) {
+    return (DayRentCost * days - ThreeDayBonus);
+  }
+
+  if ((days > 7) || (days === 7)) {
+    return (DayRentCost * days - FiveDayBonus);
+  }
```

Comment:
you can use `>=` instead of `days > 3 || days === 3` and `days > 7 || days === 7`